app/test_main.py

```diff
@@ -2,0 +1,46 @@
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mocked_valid_url() -> None:
+    with mock.patch("app.main.valid_google_url") as mock_valid_url:
+        yield mock_valid_url
+
+
+@pytest.fixture()
+def mocked_have_internet() -> None:
```

Comment:
Similar to the previous comment, the return type annotation `-> None` for the fixture `mocked_have_internet` is misleading. Consider removing the type annotation or changing it to reflect the actual return type.